### PR TITLE
Add single greytide implant item to uplink

### PR DESCRIPTION
**Pull request by arc166**
Sometimes you don't want to spend all your tc/all your TC except enough for an emag for two allies, but just a lot for one. Gives more freedom in things you can do while using this item. Buying this twice would cost more than 1 2 implant item. Also has no bullets since its supposed to be cheap option and I honestly don't get why greytide has bullets to begin with.  They're just there with no mention.

:cl:
- rscadd: Added single-use greytide implant

### CODE REVIEW

**Comment by Irradiation** [comment]
Gonna need a changelog for that one, chief.

**Comment by arc166** [comment]
You lost me, the changes are in the commits. Those are the only changes made.

**Comment by ShiftyRail** [comment]
@arc166 changelog in the body of your PR, so that it can be displayed in game.

:cl:
- rscadd: Added an option to buy a single greytide implant.

Also, why it has the beanbangs is very likely a copy & paste error.

```dm
/obj/item/weapon/storage/box/syndie_kit/greytide
	name = "box (GT)"

/obj/item/weapon/storage/box/syndie_kit/greytide/New()
	..()
	new /obj/item/weapon/implanter/traitor(src)
	new /obj/item/weapon/implanter/traitor(src)
	new /obj/item/clothing/glasses/sunglasses/sechud/syndishades(src)

/obj/item/weapon/storage/box/syndie_kit/boolets
	name = "Shotgun shells"

/obj/item/weapon/storage/box/syndie_kit/greytide/New()
	..()
	new /obj/item/ammo_casing/shotgun/fakebeanbag(src)
	new /obj/item/ammo_casing/shotgun/fakebeanbag(src)
	new /obj/item/ammo_casing/shotgun/fakebeanbag(src)
	new /obj/item/ammo_casing/shotgun/fakebeanbag(src)
	new /obj/item/ammo_casing/shotgun/fakebeanbag(src)
	new /obj/item/ammo_casing/shotgun/fakebeanbag(src)
```

`/obj/item/weapon/storage/box/syndie_kit/greytide/` should be `/obj/item/weapon/storage/box/syndie_kit/boolets/`

**Comment by arc166** [comment]
Oh I dunno how to do that.

**Comment by ShiftyRail** [comment]
Edit your original post and just add
```
:cl:
- rscadd: Added single-use greytide implant
```

**Comment by arc166** [comment]
Thanks

**Comment by jknpj** [comment]
>cheap
>still has shades

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -266,6 +266,15 @@
...
 	return
 
+obj/item/weapon/storage/box/syndie_kit/cheaptide
+	name = "box (CT)"
+
+/obj/item/weapon/storage/box/syndie_kit/cheaptide/New()
+	..()
+	new /obj/item/weapon/implanter/traitor(src)
+	new /obj/item/clothing/glasses/sunglasses/sechud/syndishades(src)
+	return
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: no trailing returns (yes I know you were just copying oldcode)

Comment (arc166) [reply code/game/objects/items/weapons/storage/uplink_kits.dm]: Fixed, assuming that means ''delete that return''.

**Pull request merged**